CHANGELOG-1.10.md

```diff
@@ -199,6 +199,8 @@ In terms of networking, Kubernetes 1.10 is about control. Users now have beta su
 
 ## Before Upgrading
 
+* If you need to downgrade from 1.10 to 1.9.x, downgrade to v1.9.6 to ensure PV and PVC objects can be deleted properly.
```

Comment:
Do we need to care about downgrading from 1.10 to 1.8.x?  With this change, we should recommend they downgrade to 1.9.6 first before 1.8.x.